lib/cisco_node_utils/vlan.rb

```diff
@@ -76,7 +76,7 @@ def cli_error_check(result, ignore_message=nil)
         (result[2]['body'].split(errors) - ['']).each_slice(2).map(&:join)
       error_list.each do |msg|
         next if ignore_message && msg.to_s.include?(ignore_message)
-        fail result[2]['body']
+        fail Cisco::CliError
```

Comment:
Did not realize this until now, but we loose granularity.  We still need to bubble the actual error message `result[2]['body'] up so that puppet users see the specific error message.